src/managers/WebhookManager.js

```diff
@@ -14,14 +14,47 @@ import logger from '../lib/logger';
 import request from 'request';
 import throttle from 'lodash/throttle';
 
+type DefaultWebhookVariables = {
+  PARTICLE_DEVICE_ID: ?string,
```

Comment:
Should these be nullable or optional?  I would think that they would be optional